src/__tests__/ActionMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import {cleanup, render as HTMLRender} from '@testing-library/react'
+import 'babel-polyfill'
+import {axe, toHaveNoViolations} from 'jest-axe'
+import React from 'react'
+import theme from '../theme'
+import {ActionMenu} from '../ActionMenu'
+import {COMMON} from '../constants'
+import {behavesAsComponent, checkExports} from '../utils/testing'
+import {ThemeProvider} from 'styled-components'
+import {BaseStyles} from '..'
+expect.extend(toHaveNoViolations)
+
+function SimpleActionMenu(): JSX.Element {
+  return (
+    <ThemeProvider theme={theme}>
+      <BaseStyles>
+        <ActionMenu
+          items={[
+            {text: 'New file'},
+            ActionMenu.Divider,
+            {text: 'Copy link'},
+            {text: 'Edit file'},
+            {text: 'Delete file', variant: 'danger'}
+          ]}
+        />
+      </BaseStyles>
+    </ThemeProvider>
+  )
+}
+
+describe('ActionMenu', () => {
+  behavesAsComponent({Component: ActionMenu, systemPropArray: [COMMON], options: {skipAs: true, skipSx: true}})
+
+  checkExports('ActionMenu', {
+    default: undefined,
+    ActionMenu
+  })
+
+  it('should have no axe violations', async () => {
+    const {container} = HTMLRender(<SimpleActionMenu />)
+    const results = await axe(container)
+    expect(results).toHaveNoViolations()
+    cleanup()
+  })
+})
```

Comment:
Following suit with a simple smoke test here - making a note to ask if we should go deeper with the tests. I have a good idea of some basic mechanics to test if we want more coverage.